python/samples/concepts/agents/step5_openai_assistant_agent_file_search.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft. All rights reserved.
+import asyncio
+import os
+
+from semantic_kernel.agents.open_ai.azure_open_ai_assistant_agent import AzureOpenAIAssistantAgent
+from semantic_kernel.agents.open_ai.open_ai_assistant_agent import OpenAIAssistantAgent
+from semantic_kernel.contents.chat_message_content import ChatMessageContent
+from semantic_kernel.contents.utils.author_role import AuthorRole
+from semantic_kernel.kernel import Kernel
+
+AGENT_NAME = "CodeRunner"
+AGENT_INSTRUCTIONS = "Run the provided code file and return the result."
+
+# Note: you may toggle this to switch between AzureOpenAI and OpenAI
+use_azure_openai = True
+
+
+# A helper method to invoke the agent with the user input
+async def invoke_agent(agent: OpenAIAssistantAgent, thread_id: str, input: str) -> None:
+    """Invoke the agent with the user input."""
+    await agent.add_chat_message(thread_id=thread_id, message=ChatMessageContent(role=AuthorRole.USER, content=input))
+
+    print(f"# {AuthorRole.USER}: '{input}'")
+
+    async for content in agent.invoke(thread_id=thread_id):
+        if content.role != AuthorRole.TOOL:
+            print(f"# {content.role}: {content.content}")
+
+
+async def main():
+    # Create the instance of the Kernel
+    kernel = Kernel()
+
+    # Add the OpenAIChatCompletion AI Service to the Kernel
+    service_id = "agent"
+
+    # Create the agent configuration
+    if use_azure_openai:
+        agent = AzureOpenAIAssistantAgent(
+            kernel=kernel, service_id=service_id, name=AGENT_NAME, instructions=AGENT_INSTRUCTIONS
+        )
+    else:
+        agent = OpenAIAssistantAgent(
+            kernel=kernel, service_id=service_id, name=AGENT_NAME, instructions=AGENT_INSTRUCTIONS
+        )
+
+    txt_file_path = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
+        "resources",
+        "agent_assistant_file_search",
+        "travelinfo.txt",
+    )
+
+    with open(txt_file_path, "rb") as file:
+        file = await agent.client.files.create(file=file, purpose="assistants")  # type: ignore
```

Comment:
are we going to include a method for this and the vector store creation?

Reply:
The consensus from the ADR review was to not include a method to do this but to target the client specifically...